README.md

```diff
@@ -63,18 +64,7 @@ A list of contributors to the project can be found here: [Contributors](https://
 
 Algebird modules are available on maven central. The current groupid and version for all modules is, respectively, `"com.twitter"` and  `0.12.2`.
 
-Current published artifacts are
-
-* `algebird-core_2.11`
-* `algebird-core_2.10`
-* `algebird-test_2.11`
-* `algebird-test_2.10`
-* `algebird-util_2.11`
-* `algebird-util_2.10`
-* `algebird-bijection_2.11`
-* `algebird-bijection_2.10`
-
-The suffix denotes the scala version.
+See [Algebird's page on the Scaladex](https://index.scala-lang.org/twitter/algebird) for information on all published artifacts and their associated Scala versions. Algebird currently supports Scala 2.10 and 2.11.
```

Comment:
they've got dropdowns for all combinations on the top right of the page... so good, and way easier to keep relevant.